solutions/mols_search/mols-search-webserver/src/indexer/index.py

```diff
@@ -30,19 +30,11 @@ def create_table(client, table_name=None, dimension=VECTOR_DIMENSION,
 
 
 def insert_vectors(client, table_name, vectors):
-<<<<<<< HEAD
-    if not client.has_collection(collection_name=table_name):
-=======
     if not client.has_collection(table_name):
```

Comment:
@shiyu22, on a side note, I noticed that some of the client api in bootcamp returns assume one return value

Isn't the `client.has_collection` supported to return 2 values : `status` and `response` ? In that case, this will always be true.

Reply:
Yes, you are right. And thanks for your contribution.